coalib/bearlib/languages/definitions/Haskell.py

```diff
@@ -0,0 +1,15 @@
+from coalib.bearlib.languages.Language import Language
+
+
+@Language
+class Haskell:
+    extensions = '.hs',
```

Comment:
Another extension for Haskell is `.lhs` which is Literate Haskell. I'm not sure. Should we add it as well?

Reply:
Literate Haskell is Haskell language nested 'within' another language (https://github.com/coala/coala/issues/1690); i.e. languages where they use a enclosing language of some sort to demarcate code, and the outer language is usually some textual content in another language.  https://wiki.haskell.org/Literate_programming has some more examples.

Literate Haskell needs to be a different language definition, describing those demarcations.